wrappers/android/tools/camera/src/main/java/com/intel/realsense/camera/Streamer.java

```diff
@@ -123,7 +126,13 @@ void configAndStart() throws Exception {
                 configStream(config);
             if(mListener != null)
                 mListener.config(config);
-            mPipeline.start(config);
+            try (PipelineProfile pp = mPipeline.start(config)) {
+                List<Sensor> sensors = pp.getDevice().querySensors();
+                for (Sensor sen : sensors) {
+                    if (sen.supports(Option.GLOBAL_TIME_ENABLED))
+                        sen.setValue(Option.GLOBAL_TIME_ENABLED, 0);
```

Comment:
Why?

Reply:
The frames are out of sync when the global time stamp is active.